addons/website_sale/views/snippets.xml

```diff
@@ -62,8 +62,30 @@
                 </table>
             </div>
 
-            <div name="style" data-no-preview="true">
-                <!-- Placeholder to be filled in JS -->
+            <we-row data-name="ribbon_options">
+                <we-select string="Ribbon" data-no-preview="true" class="o_wsale_ribbon_select">
+                    <we-button data-set-ribbon="" data-name="no_ribbon_opt">None</we-button>
+                    <!-- Ribbons are filled in JS -->
+                </we-select>
+                <we-button data-edit-ribbon="" class="fa fa-edit" data-no-preview="true" data-dependencies="!no_ribbon_opt"/>
+                <we-button data-create-ribbon="" class="fa fa-plus text-success" data-no-preview="true"/>
+            </we-row>
+            <div class="d-none" data-name="ribbon_customize_opt">
+                <we-row string="Ribbon">
+                    <we-input data-set-ribbon-html="" data-apply-to=".o_wsale_ribbon_dummy" data-no-preview="true"/>
+                    <we-button class="fa fa-check" data-save-ribbon="" title="Validate" data-no-preview="true"/>
+                    <we-button class="fa fa-trash" data-delete-ribbon="" title="Delete" data-no-preview="true"/>
+                </we-row>
+                <we-colorpicker string="Text Color" title="" data-select-style="" data-apply-to=".o_wsale_ribbon_dummy" data-css-property="color" data-no-preview="true"/>
+                <we-colorpicker string="BG Color" title="" data-select-style="" data-apply-to=".o_wsale_ribbon_dummy" data-css-property="background-color" data-color-prefix="bg-" data-no-preview="true"/>
```

Comment:
- Switch bg and text colorpickers like I explained ;) Top widgets influence bottom widgets, not the opposite when possible.
- Rename "BG Color" into "Background Color"